common/src/lib.rs

```diff
@@ -43,6 +43,7 @@ pub enum Source {
     Hello_World(HelloWorldSourceConfig),
     Excel_Connector(ExcelConfig),
     Postgres_Connector(PostgresConnectorConfig),
+    Tagging_Transformer(TaggingTransformerConfig),
```

Comment:
We might want to create a new type of Enum, rather than calling this one a "Source"

Reply:
Maybe doesn't need to be done right now, but leaving a FIXME comment would be helpful

Reply:
Oh, good idea! I'll look into where these enums are used to figure out if it'll be a quick change.